### PR TITLE
add content

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

File: help/help.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# NAME
+systemtap — programmable system-wide instrumentation system running in a container.
+
+# DESCRIPTION
+This container image provides functionality of packages `systemtap`, `systemtap-client` and `systemtap-testsuite`, which contains plenty of examples.
+
+# USAGE
+In order to use systemtap successfully, it requires debug information for running kernel, namely package `kernel-debuginfo`. There are two ways to do this when running systemtap in this container:
+
+1. Install `kernel-debuginfo` and `kernel-devel` inside container.
```
Comment (eliskasl) [inline help/help.md]: kernel-devel is apparently installed already

Comment (eliskasl) [reply help/help.md]: No package kernel-debuginfo available.

Comment (TomasTomecek) [reply help/help.md]: I'm glad you brought this up, wasn't sure if I should provide steps how to do that. Am on it!

File: README.md [hunk 2]
```diff
@@ -1,37 +1,34 @@
...
 
- * `help` - a directory containing `help.md` file. This serves as a base for container help page. It can be templated on per-version and per-os\_variant bases. On executing `make doc` in a `<version>/<os_variant>/` directory, it will search for `<version>/root/help.values` and `<version>/<os_variant>/root/help.values`, source these two scripts (if they exist) and run `envsubst` (environment variable substitution) to obtain `help.md` specific to the currently `make`-ed image.
- * `LICENSE` — pick the right license
- * `Makefile` — so it's easy to build, run, test
- * `openshift-template.yml` - general OpenShift template. A postfix template https://github.com/container-images/postfix/blob/master/openshift-template.yml
-  * update it. It includes several commands what to update
- * `README.md` — global documentation for the whole service
-  * what is it, how to build, how to use
- * `tests/` — these tests should verify that the image works
+2. script to print backtrace from a selected kernel function:
```
Comment (phracek) [inline README.md:25]: Sorry for digging. This is an example?
Does it make to include some real example like is mentioned there https://sourceware.org/systemtap/examples/?

Do you plan to create a test wether systemtap works properly?

Comment (TomasTomecek) [reply README.md:25]: Good point with the example, I'll write down that those examples you are linking are actually available inside.

Regarding tests: I tried, but it was really hard -- the biggest issue is that you need those huge 500MB packages installed in the right version, so I'll write the tests at some point.

**Comment by TomasTomecek** [comment]
@eliskasl @phracek thank you for review; I think I addressed your comments.

**Comment by eliskasl** [comment]
LGTM

**Comment by TomasTomecek** [comment]
@eliskasl @phracek thank you!

**Pull request merged**